packages/contracts/src/networks.ts

```diff
@@ -1004,7 +1004,7 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     queryFilterBlockLimit: 2000,
     actionGasLimitBuffer: true,
     useHigherMaxGasLimit: true,
-    eip2028: false,
+    eip2028: true,
```

Comment:
Since this field is no longer used for any network, I would just remove it. We should also update the `getGasPerCalldata` function so that it no longer includes logic related to this.

Reply:
Slight preference for keeping it in case there's a network in the future that doesn't support EIP-2028. Not opinionated though